_posts/2023-02-08-fsdsinet-models.md

```diff
@@ -41,7 +40,7 @@ These models make predictions each 0.5 seconds by default.
 Internally, they operate on 10 ms frames, and the number of frames to jump (50 by default) can be controlled with the `patchHopSize` parameter.
 Additionally, the models can be configured to return embeddings instead of activations by setting the parameter `output="model/global_max_pooling1d/Max"`.
 
-Finally, we can visualize the top activations using `matplotlib`:
+Finally, we can visualize the top activations using `matplotlib` and the model's [metadata](https://essentia.upf.edu/models/audio-event-recognition/fsd-sinet/fsd-sinet-vgg42-tlpf_aps-1.json):
```

Comment:
Maybe "Finally" is not the best word here. Another proposal: "The following code visualizes ...."